app/services/bibid_updater.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Service to update Voyager bibids to alma mms ids
+class BibidUpdater
+  def self.update
+    new.update
+  end
+
+  def update
+    progress_bar
+    resources.each do |resource|
+      change_set = ChangeSet.for(resource)
+      change_set.source_metadata_identifier = transform_id(change_set.source_metadata_identifier)
+      change_set_persister.save(change_set: change_set)
+      progress_bar.progress += 1
+    end
+  end
+
+  private
+
+    def change_set_persister
+      @change_set_perisister ||= ChangeSetPersister.new(metadata_adapter: metadata_adapter,
+                                                        storage_adapter: Valkyrie.config.storage_adapter)
```

Comment:
@eliotjordan should this be @change_set_persister?

Reply:
yep.